### DISCUSSION TITLE
Disabled button

### DISCUSSION THREAD

**Discussion by Esteban-DA-COSTA** [discord-jda/JDA#1739] [Questions and Help]
I'm sorry, I tried to disabled a discord action Button after I clicked it to prevent multiple actions on it but I don't know how to do it
Any help?

**Comment by DV8FromTheWorld** [comment]
You need to include your code when asking for help.
[Button#asDisabled()](https://ci.dv8tion.net/job/JDA/javadoc/net/dv8tion/jda/api/interactions/components/Button.html#asDisabled()) might be what you want tho.
